inst/startup/merger.R

```diff
@@ -1,64 +1,125 @@
 #!/usr/bin/Rscript
 args <- commandArgs(trailingOnly = TRUE)
+status <- 0
 
-# test if there is at least one argument: if not, return an error
-if (length(args) == 0) {
-  stop("At least one argument must be supplied (input file).n", call. = FALSE)
-} else if (length(args) == 1) {
-  # default output file
-  args[2] <- "out.txt"
+isError <- function(x) {
+  inherits(x, "simpleError") || inherits(x, "try-error")
 }
 
-batchJobPreparationDirectory <- args[1]
-batchTaskWorkingDirectory <- args[2]
-batchJobId <- args[3]
+getSimpleErrorMessage <- function(e) {
+  print(e$message)
+  e$message
+}
+
+getSimpleErrorCall <- function(e) {
+  deparse(e$call)
+}
 
-n <- args[4]
-numOfTasks <- args[5]
+batchTasksCount <- as.integer(args[1])
+chunkSize <- as.integer(args[2])
+errorHandling <- args[3]
+cat(chunkSize, fill = TRUE)
+cat(batchTasksCount, fill = TRUE)
+
+batchJobId <- Sys.getenv("AZ_BATCH_JOB_ID")
+batchJobPreparationDirectory <-
+  Sys.getenv("AZ_BATCH_JOB_PREP_WORKING_DIR")
+batchTaskWorkingDirectory <- Sys.getenv("AZ_BATCH_TASK_WORKING_DIR")
 
 azbatchenv <-
   readRDS(paste0(batchJobPreparationDirectory, "/", batchJobId, ".rds"))
 
+setwd(batchTaskWorkingDirectory)
+
 for (package in azbatchenv$packages) {
   library(package, character.only = TRUE)
 }
+
 parent.env(azbatchenv$exportenv) <- globalenv()
+sessionInfo()
```

Comment:
debug?

Reply:
Keeping it, I think it's useful for users to know what packages are installed / what's their current R environment